anta/custom_types.py

```diff
@@ -27,3 +27,5 @@ def aaa_group_prefix(v: str) -> str:
 Interface = Annotated[str, Field(pattern=r"^(Ethernet|Fabric|Loopback|Management|Port-Channel|Tunnel|Vlan|Vxlan)[0-9]+(\/[0-9]+)*(\.[0-9]+)?$")]
 Afi = Literal["ipv4", "ipv6", "vpn-ipv4", "vpn-ipv6", "evpn", "rt-membership"]
 Safi = Literal["unicast", "multicast", "labeled-unicast"]
+EncryptionType = Literal["RSA", "ECDSA"]
+EncryptionSize = Literal[1024, 2048, 4096]
```

Comment:
I would call this `KeySize`. Also, RSA and ECDSA have different key size options:
```bash
DC1-LEAF1A(config)#security pki key generate rsa ?
  2048  Use 2048-bit keys
  3072  Use 3072-bit keys
  4096  Use 4096-bit keys
```
```bash
DC1-LEAF1A(config)#security pki key generate ecdsa ?
  prime256v1  X9.62/SECG curve over a 256 bit prime field (P-256)
  secp384r1   NIST/SECG curve over a 384 bit prime field (P-384)
  secp521r1   NIST/SECG curve over a 521 bit prime field (P-521)
```